### PR TITLE
Add st.toHtml and st.toText functions

### LINKED ISSUE #25: [feature] Add 'toHtml' and 'toText' methods into Util.String
Suggestion:
```
function toHtml () {
		return this
			.replace( new RegExp( '\n',  'g' ), '<br />' )
			.replace( new RegExp( '"', 'g' ), '&quot;' );
};
function toText() {
		return this
			.replace( new RegExp( '<br />', 'g'), '\n' )
			.replace( new RegExp( '&quot;', 'g'), '"' );
};
```

### PR DESCRIPTION
**Pull request by gabrielpconceicao**
Closes #25. This pull request adds ```st.toHtml``` and ```st.toText``` functions.

### CODE REVIEW

**Comment by cusspvz** [comment]
Please add some test as PoC for those methods and add changes description to our `CHANGELOG.md`.

**Comment by cusspvz** [comment]
@gabrielpconceicao could you please:
* rebase from last master
* add changes you made into `CHANGELOG.md` on `Upcoming`

**Comment by gabrielpconceicao** [comment]
I have some improvements on this functions.
Check this out https://github.com/findhit/findhit/pull/1977/files [ Private ]

**Comment by cusspvz** [comment]
Can you please apply them also here?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## Upcoming
 
+## 0.2.8
```
Comment (cusspvz) [inline CHANGELOG.md]: Next time let me handle this, just place it at upcoming. ;)

**Pull request merged**